10up/publisher-media-kit
# Fix broken image paths

### Description of the Change

During the [last release](https://github.com/10up/publisher-media-kit/pull/88/commits/1a7de0d511ed848a1b46a193b82809eda90eb44f) we removed the `/assets` directory from going out in the public release and instead referenced image assets from the `/dist` directory. The problem here is that for all existing sites that use this plugin, images will now be broken and they will have to manually fix those (which is not a very intuitive process).

This PR fixes that by bringing the `/assets/images` directory back in to the release but still excluding the `/assets/css`, `/assets/js` and `/assets/svg` directories. The block markup has also been updated to account for this change.

Closes #90.

One thing to note here is that for anyone that happened to install this plugin after that release went out or who did manage to manually update the image paths, those will now reference images from the `/dist` directory. This should be fine as those images will still exist.

We can look in the future to standardize on having just the `/dist` directory but we'll need some sort of migration script in place to handle any current pages that have been set up using these blocks.

### How to test the Change

1. Install the plugin
2. Go to the newly added Media Kit page
3. Preview this page and ensure images all show correctly
4. If desired, manually change one of the images to reference the `/dist` directory instead of `/assets` and verify that image still renders

### Changelog Entry

> Fixed - Bring back the `/assets/images` directory to avoid broken images

### Credits
Props @dkotter, @kmwilkerson

### Checklist:
